ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -440,7 +442,49 @@ class RangeSelector {
         rangeSelector.options = options;
         rangeSelector.buttons = [];
 
-        rangeSelector.buttonOptions = buttonOptions;
+        const buttonOptionBindings: Record<string, {
+            title: string,
+            text: string
+        }> = {
+            month: {
+                title: chart.options.lang.rangeSelector.monthTitle,
+                text: chart.options.lang.rangeSelector.monthText
+            },
+            year: {
+                title: chart.options.lang.rangeSelector.yearTitle,
+                text: chart.options.lang.rangeSelector.yearText
+            },
+            ytd: {
+                title: chart.options.lang.rangeSelector.YTDTitle,
+                text: chart.options.lang.rangeSelector.YTDText
+            },
+            all: {
+                title: chart.options.lang.rangeSelector.allTitle,
+                text: chart.options.lang.rangeSelector.allText
+            }
+        };
```

Comment:
Do we need to create this mapping? Perhaps if we replace `YTDText` (and title counterpart) with `ytdTitle` then a few lines below, it could be simply:
```js
opt.title = chart.options.lang.rangeSelector[`${opt.type}Title`];
```